File: fizz_buzz.clj
(ns FizzBuzz.core)

(defn fizzBuzz [n]
  (cond
    (zero? (mod n (* 3 5))) "fizzbuzz"
    (zero? (mod n 5)) "buzz"
    (zero? (mod n 3)) "fizz"
    :else n))

(map fizzBuzz (range 1 101))

File: fizzBuzz.py
def fizzBuzz(n):
    if n % 15 == 0:
        print "fizzbuzz"
    elif n % 5 == 0:
        print "buzz"
    elif n % 3 == 0:
        print "fizz"
    else:
        print n

for num in range(1, 101):
    fizzBuzz(num)